tests/test_stats.py

```diff
@@ -51,3 +100,7 @@ def test_dummy_collector(self):
         stats.set_value('test', 'value', spider=self.spider)
         self.assertEqual(stats.get_stats(), {})
         self.assertEqual(stats.get_stats('a'), {})
+
+
+if __name__ == "__main__":
+    unittest_main()
```

Comment:
this looks unnecessary, tests should be selected using py.test options. I think we should be removing this code from most files.

Reply:
Sorry, tox was taking too long to run a single test :sweat_smile: 
I'll remove it, thanks!

Reply:
You can use pytest parameters with tox, e.g. `tox -e py36 -- tests/test_stats.py`

Reply:
> `tox -e py36 -- tests/test_stats.py`

I use that, it takes forever anyway. I might need to read tox docs for ways to speed things up.

Reply:
Oh, right, test collection does take a bit :sweat: